<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeleteUsersGroupMemberWin</class>
 <widget class="QWidget" name="DeleteUsersGroupMemberWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="MainBoard" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>300</width>
     <height>440</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="1,7,0">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>20</number>
      </property>
      <item>
       <widget class="QLabel" name="TisLabel">
        <property name="text">
         <string>按住Ctrl可以多选</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="MemberListWidget"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="DelButton">
          <property name="text">
           <string>踢出群组</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="CloseButton">
          <property name="text">
           <string>关闭</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
